files/build_templates/docker_image_ctl.j2

```diff
@@ -34,27 +34,16 @@ function updateSyslogConf()
     # Also update the container name
     if [[ ($NUM_ASIC -gt 1) ]]; then
         TARGET_IP=$(docker network inspect bridge --format={{ "'{{(index .IPAM.Config 0).Gateway}}'" }})
-    else
-        if [ "$CONTAINER_EXISTS" = "yes" ]; then
-            # database configuration has been synced to /etc/rsyslog.conf
-            # no need generate it to save boot time
-            return
-        fi
-        TARGET_IP="127.0.0.1"
-    fi
-    CONTAINER_NAME="$DOCKERNAME"
-    TMP_FILE="/tmp/rsyslog.$CONTAINER_NAME.conf"
-    {%- if docker_container_name == "database" %}
-    python -c "import jinja2, os; paths=['/usr/share/sonic/templates']; loader = jinja2.FileSystemLoader(paths); env = jinja2.Environment(loader=loader, trim_blocks=True); template_file='/usr/share/sonic/templates/rsyslog-container.conf.j2'; template = env.get_template(os.path.basename(template_file)); data=template.render({\"target_ip\":\"$TARGET_IP\",\"container_name\":\"$CONTAINER_NAME\"}); print(data)" > $TMP_FILE
-    {%- else %}
-    sonic-cfggen -d -t /usr/share/sonic/templates/rsyslog-container.conf.j2 -a "{\"target_ip\": \"$TARGET_IP\", \"container_name\": \"$CONTAINER_NAME\", \"platform\": \"$PLATFORM\" }"  > $TMP_FILE
-    if [ $? -ne 0 ]; then
-      echo "Error: Execute sonic-cfggen -d failed. Execute without '-d'."
-      sonic-cfggen -t /usr/share/sonic/templates/rsyslog-container.conf.j2 -a "{\"target_ip\": \"$TARGET_IP\", \"container_name\": \"$CONTAINER_NAME\", \"platform\": \"$PLATFORM\" }"  > $TMP_FILE
+        CONTAINER_NAME="$DOCKERNAME"
+        TMP_FILE="/tmp/rsyslog.$CONTAINER_NAME.conf"
+        {%- if docker_container_name == "database" %}
+        python -c "import jinja2, os; paths=['/usr/share/sonic/templates']; loader = jinja2.FileSystemLoader(paths); env = jinja2.Environment(loader=loader, trim_blocks=True); template_file='/usr/share/sonic/templates/rsyslog-container.conf.j2'; template = env.get_template(os.path.basename(template_file)); data=template.render({\"target_ip\":\"$TARGET_IP\",\"container_name\":\"$CONTAINER_NAME\"}); print(data)" > $TMP_FILE
```

Comment:
Non-blocking comment: This probably doesn't break anything by being present, but I'm guessing these variables no longer need to be present in this command, since they're being passed in via environment variables?

Reply:
Thanks for the comments. I don't have multi ASIC platform for testing, so I keep this for safety.

Reply:
I'm checking multi-ASIC KVM, and I think there's something broken. I'll confirm in a bit.